libraries/AP_HAL_ChibiOS/hwdef/include/minimize_common.inc

```diff
@@ -0,0 +1,107 @@
+# for now we make this define, but we should migrate from this to much
+#  more specific defines - added below.
+define HAL_MINIMIZE_FEATURES 1
```

Comment:
this shouldn't be in here - it should only go in minimize_features

Reply:
> this shouldn't be in here - it should only go in minimize_features

I'm pretty sure you haven't checked what that would do to these new board builds.  I'm *pretty* sure you weren't after zig-zag mode on your fpv-osd boards.  Or weathervaning.  Or guided-nogps.  Or several other weird things.

I *have* added a patch onto this PR which moves enabling/disabling turtle mode into `minimize_features.inc` - that means boards moved to `minimize_fpv_osd.inc` will now get turtle mode (assuming they meet the other prerequisites for it).

Reply:
I think we are talking at cross purposes. My goal with minimize_common was to include the maximal set of useful things that still fits. The more we get into a conversation about whether it should be in or out the more subjective it becomes. Henry feels strongly about some features, I feel strongly about others - lets include all the features that people feel strongly about and see where that gets us. I have been surprised at how much fits - it seems much of the savings can come from the elimination of esoteric drivers rather than functionality.

Reply:
I _really_ don't think we should be trying to include the maximal set of useful features that still fits into boards. That only leads to micromanaging once again. We'll end up having boards stop building every week and having to remove stuff each time as the existing features grow.

Better to simply have a set of features that people are most likely to want to use with that type of board, and if people want more then they should use the custom build server.